api/src/main/java/net/kyori/adventure/text/format/Style.java

```diff
@@ -599,6 +612,8 @@ static boolean hasAll(final @NonNull Set<Merge> merges) {
     /**
      * Sets the font.
      *
+     * <p>Supported for Minecraft: Java Edition 1.16+</p>
```

Comment:
I'm wondering if we should introduce some kind of new javadoc tag instead?`Something like `@minecraft 1.16` or such.

As far as I know, it is possible to have them render with additional text/information in the built Javadocs as well: https://docs.oracle.com/javase/8/docs/technotes/guides/javadoc/index.html

Reply:
I also felt something like that was missing when I was writing this, going to look into this!